dev-qt/qtprotobuf/qtprotobuf-0.1.1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="gRPC and Protobuf generator and bindings for Qt framework"
+HOMEPAGE="https://semlanik.github.io/qtprotobuf/"
+SRC_URI="https://github.com/semlanik/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="
+	>=dev-lang/go-1.10
+	>=dev-libs/protobuf-3.6
+	>=dev-qt/qtcore-5.12.3
+	>=dev-qt/qtnetwork-5.12.3
+	>=dev-qt/qtdeclarative-5.12.3"
+DEPEND="${RDEPEND}
+	>=dev-util/cmake-3.1"
```

Comment:
This is not needed.
If relatively new cmake was needed (e.g. newer than oldest available cmake version in gentoo.git), then that dependency would have to be in `BDEPEND`, not `DEPEND`.